flexmeasures/ui/templates/views/status.html

```diff
@@ -143,7 +94,30 @@ <h3>Latest forecasting jobs of {{ asset.name }}</h3>
                                     {% endif %}
                                 </span>
                             </td>
-                            <td class="d-none">
+                            <td class="text-right">
+                                <!-- Button -->
+                                <a href="#" class="btn btn-default btn-success" role="button" id="job-metadata-info-button" data-bs-target="#JobMetadataModal-{{ job_data['metadata_hash'] }}" data-bs-toggle="modal">
+                                    Info
+                                </a>
+                                
+                                <!-- Modal -->
+                                <div class="modal fade" id="JobMetadataModal-{{ job_data['metadata_hash'] }}" tabindex="-1" role="dialog" aria-labelledby="JobMetadataInfo" aria-hidden="true">
+                                    <div class="modal-dialog" role="document">
+                                        <div class="modal-content">
+                                            <div class="modal-header">
+                                                <h5 class="modal-title" id="JobMetadataInfo">Info</h5>
+                                                <button id="modalCloseButton" type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close" onmousedown="event.stopPropagation(); event.preventDefault();">
+                                                    <span aria-hidden="true">&times;</span>
```

Comment:
This `span` has to go, the `btn-close` class already draws an "X". If you look closely, there are now to Xs, one small (from this line) and a larger one.